workflows/rnaseq.nf

```diff
@@ -43,9 +43,9 @@ if (!params.skip_bbsplit && !params.bbsplit_index && params.bbsplit_fasta_list)
 
 // Check alignment parameters
 def prepareToolIndices  = []
-if (!params.skip_bbsplit)   { prepareToolIndices << 'bbsplit'             }
-if (!params.skip_alignment) { prepareToolIndices << params.aligner        }
-if (params.pseudo_aligner)  { prepareToolIndices << params.pseudo_aligner }
+if (!params.skip_bbsplit) { prepareToolIndices << 'bbsplit' }
+if (!params.skip_alignment) { prepareToolIndices << params.aligner }
+if (!params.skip_pseudo_alignment) { prepareToolIndices << params.pseudo_aligner }
```

Comment:
We are also doing parameter validation for `--pseudo_aligner` and `--aligner` via the Nextflow schema which should fail sooner before hitting this logic which is why I hadn't included it. But we can be explicit 👍🏽

Reply:
Done https://github.com/nf-core/rnaseq/pull/1039/commits/993fff67ba7590c85cb107143513943ff39dbd21

Reply:
> We are also doing parameter validation for `--pseudo_aligner` and `--aligner` via the Nextflow schema which should fail sooner before hitting this logic which is why I hadn't included it. But we can be explicit 👍🏽

Admittedly, those peculiarities escaped me. I just assumed that `!params.skip_pseudo_alignment` will also evaluate to `true` when no pseudoalignment is performed at all (because the user performs a regular run with alignment).